egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -298,8 +313,9 @@ if [ $stage -le -4 ] && [ -z "$egs_dir" ]; then
   extra_opts+=(--transform-dir $transform_dir)
   extra_opts+=(--left-context $left_context)
   extra_opts+=(--right-context $right_context)
-  extra_opts+=(--valid-left-context $((chunk_width + left_context)))
-  extra_opts+=(--valid-right-context $((chunk_width + right_context)))
+  extra_opts+=(--valid-left-context $((minibatch_chunk_size + left_context)))
```

Comment:
Here left_context already includes chunk_left_context and model_left_context, which is what is in the training condition. The valid_left_context seems to additionally add chunk_width frames to left_context. The reason I replace chunk_width with minibatch_chunk_size is that in chunk_width is relatively larger (e.g. 200), which makes the forward pass take much more time in diagnosis job, and causes derivative test fails in model combination stage. 
minibatch_chunk_size is the same as chunk_width by default. so if the training is not in state preserving mode, the behavior is still same as before.